### DISCUSSION TITLE
Support for Air Conditioner Mate

### DISCUSSION THREAD

**Discussion by yuntian123456** [make-all/tuya-local#2052] [Q&A]
Is it possible to support  Air Conditioner Mate?

_An air conditioner mate is a smart air conditioner control converter. It uses Wi-Fi and IR communications technologies to make a conventional air conditioner smart. It enables you to control the air conditioner's power switch and timer/delay switch remotely through TuyaSmart. It also allows you to record and view the power consumption data of high-power Appliances like air conditioners._ 

Can get more infos here:  https://support.tuya.com/en/help/_detail/K9hw4ex0lv87p

Also here is how to use:  https://github.com/xZetsubou/hass-localtuya/discussions/152#discussioncomment-9398723

More info: https://github.com/xZetsubou/hass-localtuya/issues/270#issuecomment-2185511891

Below is DPs of my device: 

          "dps": {
                "devId": "02251703e8db84******",
                "dps": {
                    "1": "send_ir",       **#control**
                    "2": -1                   **#study_code**
                    "7": -1                   **#key_study**
                    "104": 15,              **#current**
                    "105": 7,                **#power**
                    "106": 2113,          **#voltage**
                }

**Comment by make-all** [comment]
To get this supported, you would need to file a New Device request. It does look like it will return some data, so should be detectable by this integration.
